dot/services_integration_test.go

```diff
@@ -299,7 +308,8 @@ func TestNewWebSocketServer(t *testing.T) {
 	err := InitNode(cfg)
 	require.NoError(t, err)
 
-	stateSrvc, err := createStateService(cfg)
+	nodeInstance := nodeBuilder{}
```

Comment:
nit rename to `nodeBuilder` instead of `nodeInstance`?

Reply:
updated to builder